.github/workflows/vm_test.yml

```diff
@@ -55,7 +58,7 @@ jobs:
     - name: Set VM name
       id: set-vm-name
       run: |
-        echo "VM_NAME=${{ github.event.repository.name }}-pr${{ github.event.pull_request.number }}-${{ steps.generate-names.outputs.name }}" >> $GITHUB_ENV
```

Comment:
Is there a reason to change the name of the VM? The PR number was quite handy to search it.

Reply:
yes, if we wake up from a manual trigger, there is no PR number, so this variable would be empty.